trace-demo/README.md

```diff
@@ -41,31 +41,30 @@ The following instructions install the OpenTelemetry [demo application](https://
     helm repo update
     ```
 
-### Activate Kyma Tracing preview
-
-1. (Temporary) In the Telemetry operator, enable the tracing feature:
-
-    ```bash
-    kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/main/components/telemetry-operator/config/crd/bases/telemetry.kyma-project.io_tracepipelines.yaml
-    kyma deploy -s main --component telemetry --value telemetry.operator.controllers.tracing.enabled=true
-    ```
-
-2. (Temporary) Activate Istio tracing based on w3c-tracecontext and OTLP:
-    ```bash
-    kyma deploy -s main --component istio -f https://raw.githubusercontent.com/kyma-project/examples/main/trace-demo/istio-values.yaml
-    ```
-3. If necessary, restart the relevant workloads.
-
-4. To enable the Jaeger backend, create a new TracePipeline:
-   ```bash
-   kubectl apply -f https://raw.githubusercontent.com/kyma-project/examples/main/trace-demo/tracepipeline.yaml
+### Activate a Kyma TracePipeline
+
+1. Provide a tracing backend and activate it
+   Install [Jaeger in-cluster](./../jaeger/) or provide a custom backend supporting the OTLP protocol
+2. Activate the Istio tracing feature
+   To [activate Istio](https://kyma-project.io/docs/kyma/main/01-overview/main-areas/telemetry/telemetry-03-traces#step-2-enable-istio-tracing) to report span data, an Istio telemetry resource needs to get applied, setting the sampling rate to 100% (not recommended for production).
+   ```yaml
+   apiVersion: telemetry.istio.io/v1alpha1
+   kind: Telemetry
+   metadata:
+   name: tracing-default
+   namespace: istio-system
+   spec:
+   tracing:
+   - providers:
+      - name: "kyma-traces"
+   randomSamplingPercentage: 100.00
```

Comment:
The two spaces are missing. `randomSamplingPercentage`  should be on the same level as `providers`